src/models/event.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+
+const EventSchema = new mongoose.Schema({
+  key: { type: String, immutable: true, required: true, unique: true },
+  name: { type: String, required: true },
+  category: {
+    type: String,
+    enum: [
+      "corporate",
+      "social",
+      "outreach",
+      "mentoring",
+      "techTeam",
+      "generalMeeting",
+      "growth",
+      "other",
+    ],
+    required: true,
+  },
```

Comment:
are these needed?

Reply:
ahh sorry for just seeing this, but yes ideally techTeam should be renamed to explorations. growth also doesn't exist anymore, so we can remove it